package.json

```diff
@@ -18,6 +18,13 @@
   },
   "dependencies": {
     "basscss-sass": "^3.0.0",
+    "@babel/core": "^7.12.10",
```

Comment:
These deps were moved from dev to regular dependencies as I was having issues building on production (command "webpack" not found). Since we build our assets in production environments, these aren't strictly development dependencies.

It looks like webpacker has also moved these from `devDependencies` for similar reasons: https://github.com/rails/webpacker/pull/137

cc @aduth

Reply:
> These deps were moved from dev to regular dependencies as I was having issues building on production (command "webpack" not found). Since we build our assets in production environments, these aren't strictly development dependencies.
> 
> It looks like webpacker has also moved these from `devDependencies` for similar reasons: [rails/webpacker#137](https://github.com/rails/webpacker/pull/137)
> 
> cc @aduth

Seems okay 👍  We had similar need in https://github.com/18F/identity-site/pull/468#discussion_r538839390